src/Maestro/Maestro.Web/Api/v2019_01_16/Models/Build.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using JetBrains.Annotations;
+
+namespace Maestro.Web.Api.v2019_01_16.Models
+{
+    public class Build : Maestro.Web.Api.v2018_07_16.Models.Build
+    {
+        public Build([NotNull] Data.Models.Build other) : base(other)
+        {
+            if (other == null)
+            {
+                throw new ArgumentNullException(nameof(other));
+            }
+
+            GitHubRepository = other.GitHubRepository;
+            GitHubBranch = other.GitHubBranch;
+            AzureDevOpsBuildId = other.AzureDevOpsBuildId;
+            AzureDevOpsBuildDefinitionId = other.AzureDevOpsBuildDefinitionId;
+            AzureDevOpsAccount = other.AzureDevOpsAccount;
+            AzureDevOpsProject = other.AzureDevOpsProject;
+            AzureDevOpsBuildNumber = other.AzureDevOpsBuildNumber;
+            AzureDevOpsRepository = other.AzureDevOpsRepository;
+            AzureDevOpsBranch = other.AzureDevOpsBranch;
+        }
+
+        public int AzureDevOpsBuildId { get; set; }
+
+        public int AzureDevOpsBuildDefinitionId { get; set; }
+
+        public string AzureDevOpsAccount { get; set; }
```

Comment:
Can you move these into AzureDevOpsInfo and GitHubInfo properties so the properties are grouped correctly.